File: app_kotlin/src/main/java/com/cxsplay/appkotlin/net/RHelper.java
package com.cxsplay.appkotlin.net;


import retrofit2.Retrofit;

/**
 * Created by CxS on 2018/4/12
 */
public class RHelper {

    private static ApiService apiService;

    static {
        Retrofit retrofit = RUtil.getRetrofitBuilder("").build();
        apiService = retrofit.create(ApiService.class);
    }

    public static ApiService getApiService() {
        return apiService;
    }
}

File: app_kotlin/src/main/java/com/cxsplay/appkotlin/netk/RHelper.kt
package com.cxsplay.appkotlin.netk

/**
 * Created by CxS on 2019/5/21
 */
object RHelper {

    var apiService: ApiService? = null
        private set

    init {
        val retrofit = RUtil.getRetrofitBuilder("http://192.168.1.104:8080").build()
        apiService = retrofit.create(ApiService::class.java)
    }
}
